runtime/js/99_main.js

```diff
@@ -294,7 +294,12 @@ async function pollForMessages() {
     );
   }
   while (!isClosing) {
-    const data = await op_worker_recv_message();
+    const recvMessage = op_worker_recv_message();
+    if (globalThis[messagePort.unrefPollForMessages] === true) {
```

Comment:
Maybe use stored `globalThis` reference? I think this will crash if user does `delete globalThis`?

Reply:
user cannot do that right?

```
error: Uncaught SyntaxError: Delete of an unqualified identifier in strict mode.
delete globalThis;
       ^
    at <anonymous> (file:///Users/sr/c/denoland/deno/a.js:1:8)
```